### PR TITLE
fix(ui): Fix command palette

### PR DESCRIPTION
**Pull request by billyvg**
`react-keydown` needs a root DOM el in order to work, `React.Fragment` doesn't cut it

### CODE REVIEW

File: src/sentry/static/sentry/app/views/app.jsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const App = createReactClass({
     if (Object.keys(newState).length > 0) this.setState(newState);
   },
 
-  @keydown('cmd+shift+p')
+  @keydown('meta+shift+p', 'meta+k')
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/app.jsx]: trying this, less interference with chrome dev tools (its slack's switcher hot key)

Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/app.jsx]: hmmm, I like it.

Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/app.jsx]: what's the difference between cmd and meta? 

docs weren't 100% clear to me
>`meta` (aka `command` or `cmd`)

**Review by MaxBittker — APPROVED** [review]
lgtm 👍

**Pull request merged**